galaxy_ng/app/tasks/deletion.py

```diff
@@ -91,7 +91,7 @@ def delete_container_distribution(instance_ids):
     general_multi_delete(instance_ids=instance_ids)
 
     log.info("Running orphan_cleanup to delete Container objects and artifacts")
-    orphan_cleanup(content_pks=None, orphan_protection_time=0)
+    orphan_cleanup(content_pks=None, orphan_protection_time=10)
```

Comment:
is `10`a reasonable number to protect? is this number in seconds?

Reply:
The value is in minutes.  The default setting in pulpcore is [here](https://github.com/pulp/pulpcore/blob/main/pulpcore/app/settings.py#L251).